### ISSUE TITLE
display or link to node documentation

### ISSUE THREAD

**Issue by clstaudt** [logspace-ai/langflow#247]
As a langflow user, I do not see a quick and easy way to look up the documentation for a node in my workflow.

![Screenshot 2023-05-04 at 10 51 24](https://user-images.githubusercontent.com/875194/236156753-dee68ed5-63b1-46ec-8b0c-0d3f71a9d556.png)

For example, the node could show a tooltip on hover or have a link to open the corresponding `langchain` class documentation.

Furthermore, it would be great if more documentation was attached to the inputs and outputs of a node. Not just the type, but a description text of the parameters.

In short, where are the docstrings? 

I realize this also depends on documentation quality in `langchain`, which is evolving and was not in the greatest shape last time I checked. Then again, if your project is about LLMs, it would look good on it to have great docstrings, because they can mostly be auto-generated from the code nowadays...

**Comment by ogabrielluiz** [comment]
Hey, @clstaudt.

I agree with you. A link to LangChain would be interesting but would require a lot of manual labor which is what we are trying to avoid so that API changes to break everything all the time.

Maybe we can think of better ways to explain things in the node. The hover data on the inputs and outputs are really the types that are checked for when connecting nodes, but I'm sure we can think of something better.

**Comment by clstaudt** [comment]
Assuming `langchain` classes have good docstrings, shouldn't it be possible to just extract and display them in the UI?

```python
import inspect

class MyClass:
    """
    This is the MyClass docstring.
    """

    def method1(self):
        """
        This is the method1 docstring.
        """
        pass

    def method2(self):
        """
        This is the method2 docstring.
        """
        pass

def extract_docstrings(cls):
    docstrings = {}
    docstrings['class'] = inspect.getdoc(cls)
    
    for name, obj in inspect.getmembers(cls, predicate=inspect.isfunction):
        docstrings[name] = inspect.getdoc(obj)
    
    return docstrings

docstrings = extract_docstrings(MyClass)

for key, value in docstrings.items():
    print(f"{key}:\n{value}\n")
```

**Comment by airtonix** [comment]
> Hey, @clstaudt.
> 
> I agree with you. A link to LangChain would be interesting but would require a lot of manual labor which is what we are trying to avoid so that API changes to break everything all the time.
> 
> Maybe we can think of better ways to explain things in the node. The hover data on the inputs and outputs are really the types that are checked for when connecting nodes, but I'm sure we can think of something better.

Easy way to fix this: 

1. work on a node requires working user documentation
2. user documentation can come in the form of module comments
3. node documentation is published when you do a prod release
4. node documentation is published to a static website in separate pages that match the node name
5. node documentation should have a version selector next to it.
6. within the langflow ui, the `( ? )` icon should link to to something like: `https://langflow.docs/node/chat-openai?version=0.7.5`

**Comment by ogabrielluiz** [comment]
@clstaudt This is what we do. The problem is that documentation is not just a docstring.

**Comment by clstaudt** [comment]
Stale bot - what is it good for?

**Comment by lucaseduoli** [comment]
Function already implemented on latest version.
